src/components/historicalUsageChart/historicalUsageChart.tsx

```diff
@@ -458,11 +458,15 @@ class HistoricalUsageChart extends React.Component<
               datum.current.charts.map((chart, index) => {
                 return this.getChart(chart, index);
               })}
-            <VictoryAxis label={xAxisLabel} style={chartStyles.axis} />
+            <VictoryAxis
+              label={xAxisLabel}
+              style={chartStyles.xAxis}
+              tickValues={[1, 15, 31]}
```

Comment:
same comment as before about statically defining the ticks.

Reply:
My thought was if February has 28/29 days, and January was the previous month, the max would be 31. If February is the current month, the max is still 31.

    January - 31 days
    February - 28 days in a common year and 29 days in leap years
    March - 31 days
    April - 30 days
    May - 31 days
    June - 30 days
    July - 31 days
    August - 31 days
    September - 30 days
    October - 31 days
    November - 30 days
    December - 31 days

I suppose if we were to show only one month, then it should be dynamic.

Reply:
Ok, I've added code to make the end date dynamic. The default is 31, since that's the range (1-31 days) we intend to show even with zero data. However, if we only have one month to show (i.e., February), the end date could be shown as 28/29.